docs/configuration.md

```diff
@@ -753,7 +753,7 @@ interpretation of the options.
       no
     </td>
     <td>
-      Duration for which a signed URL should be valid.
+      Duration in minutes for which a signed URL should be valid.  When omitted, this defaults to 20 minutes.
```

Comment:
The example here is actually wrong, it gives a value of "3000" which actually ends up getting ignored and the default is used. This value really should be passed as a duration string which the golang duration parser understands OR the code should be updated to use minutes when only an integer is given.